src/main/java/domain/LadderResult.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LadderResult {
+
+    private final Map<Position, Position> firstAndLastPosition;
+
+    public LadderResult(Ladder ladder, int participantsCount) {
+        firstAndLastPosition = new LinkedHashMap<>();
+        initPosition(participantsCount);
+        calculatePosition(ladder);
+    }
+
+    private void initPosition(int participantsCount) {
+        for (int i = 0; i < participantsCount; i++) {
+            firstAndLastPosition.put(new Position(i), new Position(i));
+        }
+    }
+
+    private void calculatePosition(Ladder ladder) {
+        for (int floor = 0; !ladder.isFinish(floor); floor++) {
```

Comment:
부정연산자는 가독성을 떨어뜨린다는 의견이 있어, 저는 이런 경우 반대 메서드를 하나 더 만듭니다.
(뭔가 진행중임을 뜻하는 것도 좋은거 같은데 이것저것 붙여봐도 찰떡같지 않아 notFinish로 만족했습니다)
```suggestion
        for (int floor = 0; ladder.notFinish(floor); floor++) {
```

```java
public class Ladder {
...
   public boolean notFinish(Floor floor) {
       return !isFinish(floor);
   }
...
}
```